db/builder.cc

```diff
@@ -309,6 +310,11 @@ Status BuildTable(
       meta->file_checksum_func_name = file_writer->GetFileChecksumFuncName();
       file_checksum = meta->file_checksum;
       file_checksum_func_name = meta->file_checksum_func_name;
+      // Set unique_id
+      if (!tboptions.db_id.empty() && !tboptions.db_session_id.empty()) {
+        s = GetSstUniqueId(tboptions.db_id, tboptions.db_session_id,
```

Comment:
My preference is using "internal" IDs in the manifest. If for some reason we want the external version, we can do the conversion as needed. My main motivation is for unifying the next generation of internal unique IDs (#9058) with base cache keys. That's easier if the internal moving pieces just stick with internal IDs and only convert to external as appropriate.